cmd/fluent-manager/Dockerfile

```diff
@@ -1,5 +1,6 @@
 # Build the manager binary
-FROM ghcr.io/verrazzano/verrazzano-base:v1.0.0-20230626111229-3d949a4 as builder
+ARG BASE_IMAGE=ghcr.io/verrazzano/verrazzano-base:v1.0.0-20230727050514-6b5fb42
```

Comment:
The latest image should be supplied as an argument, rather than using a fixed image in the Dockerfile